src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -347,8 +380,16 @@ export const VenueLandingPage: React.FunctionComponent<VenueLandingPageProps> =
       <AuthenticationModal
         show={isAuthenticationModalOpen}
         onHide={closeAuthenticationModal}
+        headerMessage={"You need an account to join this party"}
```

Comment:
You can use strings directly without wrapping them in `{` `}`
```suggestion
        headerMessage="You need an account to join this party"
```